code/_onclick/adjacent.dm

```diff
@@ -83,9 +83,14 @@ Quick adjacency (to turf):
 /atom/movable/Adjacent(var/atom/neighbor)
 	if(neighbor == loc) return 1
 	if(!isturf(loc)) return 0
+	var/iter_count = 0 // Checks whether locs actually contains turfs
```

Comment:
What exactly is the issue here? I've not noticed any problems picking up items. 
What's this code doing?

Reply:
I’ve been running into a weird issue where the locs var contains a reference to items instead of their locs. I VVed and have a screenshot of this, but I don’t have access to that or my workspace until 3 hours from now.

This might be related to the BYOND version I’m on.

Reply:
I’ve been running into a weird issue where the locs var contains a reference to items instead of their locs. I VVed and have a screenshot of this, but I don’t have access to that or my workspace until 3 hours from now.

This might be related to the BYOND version I’m on.

Reply:
Please find out which version that is. If its a byond bug, we need to report it.

Reply:
Is there any updates on this?